src/webserver/webserver.c

```diff
@@ -418,16 +418,19 @@ void http_init(void)
 
 	// Get maximum number of threads for webserver
 	char num_threads[16] = { 0 };
-	if(config.webserver.threads.v.ui == 0)
+	unsigned int threads = config.webserver.threads.v.ui;
+	if(threads == 0)
 	{
 		// For compatibility with older versions, set the number of
 		// threads to the default value (50) if it was 0. Before Pi-hole
 		// FTL v6.0.4, the number of threads was computed in dependence
 		// of the number of CPUs available. This is no longer the case.
-		config.webserver.threads.v.ui = 50;
+		threads = 50;
```

Comment:
I probably just don't understand how c works here - but is the declared `unsigned int threads` just a pointer to `config.webserer.threads.v.ui`? And if so, does setting `threads` also set the config?

Reply:
No, `threads` is a local copy of the value stored in `config.webserer.threads.v.ui`, we are not working with pointers here. Hence, we can now set the value of `threads` to anything and this won't modify `config.webserer.threads.v.ui` in any way.

Reply:
The latter